ligo/src/registryDAO.mligo

```diff
@@ -0,0 +1,190 @@
+// SPDX-FileCopyrightText: 2020 TQ Tezos
+// SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+#include "types.mligo"
+#include "proposal.mligo"
+#include "registryDAO/types.mligo"
+
+(*
+ * This is a dummy entrypoint so that this file can be called with LIGO's `compile-storage` command as shown below, providing admin and token addresses.
+ *
+ * ligo compile-storage ligo/src/registryDAO.mligo dummy 'default_registry_DAO_full_storage(("tz1QozfhaUW4wLnohDo6yiBUmh7cPCSXE9Af" : address), ("tz1QozfhaUW4wLnohDo6yiBUmh7cPCSXE9Af" : address), 0n, 0n, 0n, 0n, 0n)'
+ *)
+let dummy (action, store : parameter * full_storage) : operation list * full_storage
+  = (([] : operation list), store)
```

Comment:
Yes, it works. Changed.

Reply:
I'm not sure I understand the problem with tests, wouldn't we be able to have different version of `defaultStorage` for different tests?

Or is the problem that we're going to need too many of them because the tests need a lot of different ones?